### DISCUSSION TITLE
Support request: Support ZSH git plugin

### DISCUSSION THREAD

**Discussion by fordneild** [aws/codewhisperer-command-line-discussions#95] [Support ticket]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

running git commands via the oh-my-zsh git plugin should produce suggestions.

### Actual behaviour

There is no auto complete for these git shortcuts

### Steps to reproduce

`gco ` should produce the same suggestion as git checkout. How can i revert to before the cw migration?

**Comment by fordneild** [comment]
My `justfile` auto complete it behaving differently too. Steps to go back to old fig would be appreciated. I cannot afford to have all my dev workflows broken today haha

**Comment by grant0417** [comment]
Is this an alias you had setup in Fig? If we do not support the dotfiles or plugins product, see: https://fig.io/blog/post/fig-is-sunsetting
